components/GraphPane/GraphPane.scss

```diff
@@ -16,10 +16,8 @@
   margin-top: 1rem;
 }
 
-#report-section {
-  // Hide empty sections
-  h3:has(+ h3),
-  h3:last-child {
-    display: none;
-  }
+// Hide empty sections
+.report-section hr:nth-last-child(2),
+.report-section h3:nth-last-child(1) {
+  display: none;
```

Comment:
It seems that there's no _React way_ to do this — which is kind of expected: children are rendered only if they're specified by the parent, but if the parent returns `null` then the children will not attempt to render. ♻️